content/en/tracing/setup/dotnet-core.md

```diff
@@ -217,14 +217,6 @@ The .NET Tracer can instrument the following libraries automatically:
 
 Don’t see your desired frameworks? Datadog is continually adding additional support. [Check with the Datadog team][6] for help.
 
-## Manual Instrumentation
```

Comment:
Seems a little weird to me to have a "Tracing your .NET Applications" document with an "Automatic Instrumentation" section but no "Manual Instrumentation" sections (even if it's only a link to the custom instrumentation page).

But if we do remove this section from the page, we should also remove it from `content/en/tracing/setup/dotnet-framework.md`

Reply:
Removed from `framework` page as well and added a link below integrations to the custom instrumentation page.